pycaret/tests/test_time_series_splitters.py

```diff
@@ -50,6 +50,43 @@ def test_setup_initialization(fold, fh, fold_strategy, load_data):
         assert exp_name.fold_generator.step_length == fh  # Since fh is an int
 
 
+def test_setup_pass_cv_object(load_data):
+
```

Comment:
I think we could add docstrings to tests to get a general idea of what we want to test.